<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_products_window</class>
 <widget class="QWidget" name="add_products_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>PRODUKTY</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_categories">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="box_products">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_approve">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zatwierd≈∫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_discard">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_remove">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Liczba</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
